cairo_zero/kakarot/library.cairo

```diff
@@ -3,6 +3,7 @@
 %lang starknet
 
 from openzeppelin.access.ownable.library import Ownable
+from openzeppelin.security.reentrancyguard.library import ReentrancyGuard
```

Comment:
For reference https://github.com/OpenZeppelin/cairo-contracts/blob/v0.6.1/src/openzeppelin/security/reentrancyguard/library.cairo